RadarSDK/Radar.h

```diff
@@ -239,6 +239,8 @@ typedef void (^_Nonnull RadarRouteMatrixCompletionHandler)(RadarStatus status, R
  */
 @interface Radar : NSObject
 
+@property (class, readonly) NSString *sdkVersion;
```

Comment:
@jrtibbetts Love the idea of exposing this for customers to add to their own logging.

@jsani-radar @phil-brown Should we do the same on Android for consistency?